ratis-server/src/main/java/org/apache/ratis/server/raftlog/segmented/SegmentedRaftLogWorker.java

```diff
@@ -374,6 +385,16 @@ private void flushWrites() throws IOException {
     }
   }
 
+  private void flushToPersistentStore() throws IOException {
+    if (shouldFlush() &&
+            lastFlushTimestamp.elapsedTime().compareTo(TimeDuration.valueOf(3, TimeUnit.SECONDS)) > 0) {
```

Comment:
Can we make this flush interval configurable as well?

Reply:
sure, I will make it.